src/leaflike/server.clj

```diff
@@ -0,0 +1,19 @@
+(ns leaflike.server
+  (require [org.httpkit.server :as httpkit]))
+
+(defn app [req]
+  {:status  200
+   :headers {"Content-Type" "application/json"}
+   :body    "Welcome to Leaflike"})
+
+(defonce server (atom nil))
+
+(defn start-server []
```

Comment:
The idomatic way to name functions in Clojure is to assume that people using your function will use it with a fully qualified namespace or an alias. So it is recommended to name your functions `start` and `stop` to avoid redundancy when using it (`leaflike.server/start`).

Reply:
I would add a `!` to this function name, since it is calling a function with a `!`.

Reply:
@namratachaudhary FYI, people use `!` after a function name as a convention to denote that the function is impure (i.e it causes side effects).